linux_os/guide/system/permissions/partitions/mount_option_tmp_noexec/tests/separate.fail.sh

```diff
@@ -1,15 +1,10 @@
 #!/bin/bash
 # profiles = xccdf_org.ssgproject.content_profile_C2S
 
-# Remediating would mount /tmp, which would break the test environment.
-# remediation = none
```

Comment:
I guess `remediation = none` here is essential. Unless it gives the error @yuumasato is facing

Reply:
Actually I'm not facing any issue in SSG test suite running something like:
`$python test_suite.py rule --remediate-using bash --libvirt qemu:///session rhel79 --datastream ../build/ssg-rhel7-ds.xml mount_option_tmp_noexec`
```
Setting console output to log level INFO
INFO - The DataStream contains 2 Benchmarks
INFO - 0 - scap_org.open-scap_cref_ssg-rhel7-xccdf-1.2.xml
INFO - 1 - scap_org.open-scap_cref_ssg-rhel7-pcidss-xccdf-1.2.xml
INFO - Selected Benchmark is 0
INFO - To select a different Benchmark, use --xccdf-id-number option.
INFO - The base image option has not been specified, choosing libvirt-based test environment.
INFO - Logging into /home/ggasparb/workspace/github/content/tests/logs/rule-custom-2020-04-14-1736/test_suite.log
INFO - xccdf_org.ssgproject.content_rule_mount_option_tmp_noexec
/INFO - Script runtime.pass.sh using profile xccdf_org.ssgproject.content_profile_C2S OK
INFO - Script separate.fail.sh using profile xccdf_org.ssgproject.content_profile_C2S OK
INFO - Script fstab.fail.sh using profile xccdf_org.ssgproject.content_profile_C2S OK
```

Reply:
@ggbecker Could you try without `--remediate-using bash`?

Reply:
The `oscap` remediation performs fix during the scan, so it may interfere with the scan report. OTOH, `bash` remediation executes a bash script after the initial scan finishes. So I think that this explains what's going on.

Reply:
Yes, I haven't realized this, remediations causes that `/tmp` is re-mounted so the files stored in the `/tmp` during remediation are shadowed, good catch @yuumasato 

I will return back the `remediation = none`.